hildon/Makefile.am

```diff
@@ -98,6 +106,12 @@ libhildon_@API_VERSION_MAJOR@_built_public_headers  = \
 libhildon_@API_VERSION_MAJOR@_includedir = \
 		$(includedir)/$(PACKAGE)-$(API_VERSION_MAJOR)/$(PACKAGE)
 
+if USE_MAEMO_GTK
+HEADERS_COMPAT =
+else
+HEADERS_COMPAT = maemo-gtk-compat.h
+endif
+
```

Comment:
There's no need to duplicate this condition.
Just put "HEADERS_COMPAT =" and "HEADERS_COMPAT = maemo-gtk-compat.h" to the above.